openedx/core/djangoapps/course_groups/tests/test_views.py

```diff
@@ -896,9 +902,32 @@ def test_only_unknown(self):
             expected_added=[],
             expected_changed=[],
             expected_present=[],
+            expected_preregistered=[],
             expected_unknown=usernames
         )
 
+    def test_preregistered_users(self):
+        """
+        Verify that email addresses can be preregistered for a cohort if the user associated with that email
+        address does not yet exist.
+        """
+        email_addresses = ["email@example.com", "email2@example.com", "email3@example.com"]
+        response_dict = self.request_add_users_to_cohort(
```

Comment:
I think my changes in the views.py still return email addresses:
https://github.com/edx/edx-platform/pull/15298/files#diff-27989e37c0d2923a601b01beb5186154R310